Sources/Queues/QueueContext.swift

```diff
@@ -37,4 +37,19 @@ public struct QueueContext {
         self.logger = logger
         self.eventLoop = eventLoop
     }
+
+    /// The `JobsQueue` object
```

Comment:
Looks like this comment is outdated. I'm guessing you just copied it from the request extension so that's our fault haha. Would you mind fixing it here at least?

Something like this seems fine:

```
/// Returns the default job `Queue`. 
```

Reply:
:+1: Done